scipy/optimize/_zeros_py.py

```diff
@@ -1148,7 +1146,7 @@ def start(self, f, a, b, args=()):
 
         if np.sign(fb) * np.sign(fa) > 0:
             raise ValueError("f(a) and f(b) must have different signs, but "
-                             "f({:e})={:e}, f({:e})={:e} ".format(a, fa, b, fb))
+                             f"f({a:e})={fa:e}, f({b:e})={fb:e} ")
```

Comment:
Is the alignment here correct?

Reply:
Ah no, good spot 👍

Reply:
On second thought I think this is correct.